README.md

```diff
@@ -1,2 +1,40 @@
 # cmssw_vscode
 Recipies to develop CMSSW using VSCode
+
+## Indexing files
+Indexing files allows VSCode to easily resolve references to imported files. This provides:
+- errors and warnings detection
+
+    ![Errors and warnings detection](./static/errors_and_warnings.gif "Errors and warnings detection")
+- go to definition
+
+    ![Go to definition](./static/go_to_def.gif "Go to definition")
+
+- find references
+
+    ![Find references](./static/find_references.gif "Find references")
+
```

Comment:
I would move them up as in my opinion this is mostly used feature, error and warnings detection is not so crucial here

Reply:
Done.